src/module.js

```diff
@@ -22,9 +22,9 @@ function createModule (name, testEnvironment, modifiers) {
   const skip = (parentModule !== null && parentModule.skip) || modifiers.skip;
   const todo = (parentModule !== null && parentModule.todo) || modifiers.todo;
 
-  const env = {};
+  let env = {};
   if (parentModule) {
-    extend(env, parentModule.testEnvironment);
+    env = Object.create(parentModule.testEnvironment || {});
```

Comment:
We can't use `??` here because of IE support, right?

Reply:
Newer syntax is generally fine. We use `...` rest/spread and ES6 classes for example as well. These are transpiled by Babel.

Where we're limited is calling to APIs such as new Array methods, new globals like Promise or Map, things like that.